servers/services/src/main/java/org/projectnessie/services/impl/TreeApiImpl.java

```diff
@@ -236,18 +236,21 @@ public LogResponse getCommitLog(String namedRef, CommitLogParams params)
                   if (commit.getParentHash() != null) {
                     logEntry.parentCommitHash(commit.getParentHash().asString());
                   }
-                  Objects.requireNonNull(commit.getOperations())
-                      .forEach(
-                          op -> {
-                            ContentKey key = ContentKey.of(op.getKey().getElements());
-                            if (op instanceof Put) {
-                              Content content = ((Put<Content>) op).getValue();
-                              logEntry.addOperations(Operation.Put.of(key, content));
-                            }
-                            if (op instanceof Delete) {
-                              logEntry.addOperations(Operation.Delete.of(key));
-                            }
-                          });
+                  List<org.projectnessie.versioned.Operation<Content>> operations =
+                      commit.getOperations();
+                  if (operations != null) {
```

Comment:
nit: no need to create a new variable, just use `commit.getOperations()`

Reply:
done